fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala

```diff
@@ -1,9 +1,9 @@
-package sqs
+package fs2.aws.sqs
 
 import scala.concurrent.duration.*
 
 case class SqsConfig(
     queueUrl: String,
     pollRate: FiniteDuration = 3.seconds,
-    fetchMessageCount: Int = 100
+    fetchMessageCount: Int = 10
```

Comment:
can you put link in the comment to the aws documentation with the sqs limits?

Reply:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html

Quote:

> Retrieves one or more messages (up to 10), from the specified queue.

Reply:
sorry i mean into the code comment.

Reply:
@semenodm Ah. Got it done.